lib/workers/repository/config-migration/branch/migrated-data.spec.ts

```diff
@@ -100,28 +101,29 @@ describe('workers/repository/config-migration/branch/migrated-data', () => {
     it('Migrate a JSON5 config file', async () => {
       mockedFunction(detectRepoFileConfig).mockResolvedValueOnce({
         configFileName: 'renovate.json5',
+        configFileRaw: rawNonMigratedJson5,
       });
-      mockedFunction(readLocalFile).mockResolvedValueOnce(rawNonMigratedJson5);
       MigratedDataFactory.reset();
       await expect(MigratedDataFactory.getAsync()).resolves.toEqual(
         migratedDataJson5
       );
     });
 
-    it('Returns nothing due to fs error', async () => {
-      mockedFunction(detectRepoFileConfig).mockResolvedValueOnce({
-        configFileName: undefined,
-      });
-      mockedFunction(readLocalFile).mockRejectedValueOnce(null);
+    it('Returns nothing due to detectRepoFileConfig throwing', async () => {
+      const message = 'error-message';
+      mockedFunction(detectRepoFileConfig).mockRejectedValueOnce({ message });
```

Comment:
normally we prefer Error class for rejected value. is that possible?

Reply:
test updated. 
the only thing that can throw in `detectRepoFileConfig` is `readLocalFile`, dont know if this can happen in practice though.